packages/i18n/package.json

```diff
@@ -20,6 +20,7 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"sideEffects": false,
```

Comment:
I'm not sure this one is sideEffects free (it contain globals)

Reply:
Happy to remove it to move forward 👍